Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -234,18 +225,12 @@ namespace Editor
     EditorQtApplication::EditorQtApplication(int& argc, char** argv)
         : AzQtApplication(argc, argv)
         , m_stylesheet(new AzQtComponents::O3DEStylesheet(this))
-        , m_idleTimer(new QTimer(this))
     {
-        m_idleTimer->setInterval(UninitializedFrequency);
-
         setWindowIcon(QIcon(":/Application/res/o3de_editor.ico"));
 
         // set the default key store for our preferences:
         setApplicationName("O3DE Editor");
 
-        connect(m_idleTimer, &QTimer::timeout, this, &EditorQtApplication::maybeProcessIdle);
```

Comment:
Without the idle timer and this call, maybeProcessIdle will not be called unless calling EnableOnIdle. So basically there won't be any Idle processing?

Reply:
Yeah, that's right. But [`CryEditMain` calls it directly](https://github.com/o3de/o3de/blob/c3c79b25c21f5590a6b4aad6ba4935ffe3e26abf/Code/Editor/CryEdit.cpp#L4156), right before calling `QApplication::exec()`. So this seems OK to me.

Reply:
CryEditMain calls it directly once on start. I don't see anything else doing idle processing other than the idle timer, which got removed by this change. Do you see maybeIdleProcessing get called periodically after this change?
Is this intentional? I didn't know well enough about the Cry system to judge if this is ok to do, but in general speaking for any program idle update is an important piece.